Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM python:3.11.0a3
-LABEL maintainer="ms-github@256bit.org"
+FROM python:3.11
+LABEL maintainer="operations@vico-research.com"
```

Comment:
Just remove the maintainer reference.
Probably updating to python 3.13 is something we can do in this step (but not mandatory from review perspective)